cpp/src/slice2cpp/Gen.cpp

```diff
@@ -1371,7 +1371,7 @@ Slice::Gen::ForwardDeclVisitor::visitConst(const ConstPtr& p)
     if (!isConstexprType(p->type())) // i.e. string or wstring
     {
         // The string/wstring constructor can throw, which produces a clang-tidy lint for const or static objects.
-        H << " // NOLINT(cert-err58-cpp,modernize-raw-string-literal)";
```

Comment:
There should be a comma between lints.
We do this correctly everywhere else I could find.